install/HyperlinkSidebarTheme.en.wxl

```diff
@@ -0,0 +1,70 @@
+  <String Id="InstallOptionsButton">C&amp;ustomize</String>
+  <String Id="InstallInstallButton">&amp;Install</String>
+  <String Id="InstallCloseButton">&amp;Close</String>
+  <String Id="OptionsHeader">Setup options</String>
+  <String Id="OptionsLocationLabel">Install location:</String>
+  <String Id="OptionsBrowseButton">&amp;Browse</String>
+  <String Id="OptionsOkButton">&amp;OK</String>
+  <String Id="OptionsCancelButton">&amp;Cancel</String>
+  <String Id="ProgressHeader">Setup progress</String>
+  <String Id="ProgressLabel">Processing:</String>
+  <String Id="OverallProgressPackageText">Initializing...</String>
+  <String Id="ProgressCancelButton">&amp;Cancel</String>
+  <String Id="ModifyHeader">Modify setup</String>
+  <String Id="ModifyRepairButton">&amp;Repair</String>
+  <String Id="ModifyUninstallButton">&amp;Uninstall all</String>
+  <String Id="ModifyCloseButton">&amp;Close</String>
+  <String Id="SuccessRepairHeader">Repair completed successfully.</String>
+  <String Id="SuccessUninstallHeader">Uninstall completed</String>
+  <String Id="SuccessInstallHeader">Installation completed successfully.</String>
+  <String Id="SuccessInstallMessage">Installation completed successfully!</String>
+  <String Id="SuccessHeader">Setup successful</String>
+  <String Id="SuccessLaunchButton">&amp;Launch ID-card utility</String>
+  <String Id="SuccessRestartText">You must restart your computer before you can use the software.</String>
+  <String Id="SuccessRestartButton">&amp;Restart</String>
+  <String Id="SuccessCloseButton">&amp;Close</String>
+  <String Id="FailureHeader">Setup failed</String>
+  <String Id="FailureInstallHeader">Installation failed</String>
+  <String Id="FailureUninstallHeader">Uninstall failed</String>
+  <String Id="FailureRepairHeader">Repair failed</String>
+  <String Id="FailureHyperlinkLogText">Operation failed. You can find help for installing the software on the id.ee website.</String>
+  <String Id="FailureHyperlinkLogLink">&lt;a href="#"&gt;For more information see the log file&lt;/a&gt;.</String>
+  <String Id="FailureRestartText">You must restart your computer to complete the rollback of the software.</String>
+  <String Id="FailureRestartButton">&amp;Restart</String>
+  <String Id="FailureCloseButton">&amp;Close</String>
+  <String Id="FilesInUseHeader">Files in use</String>
+  <String Id="FilesInUseLabel">The following applications are using files that need to be updated:</String>
+  <String Id="FilesInUseCloseRadioButton">Close the &amp;applications and attempt to restart them.</String>
+  <String Id="FilesInUseDontCloseRadioButton">&amp;Do not close the applications. A reboot will be required.</String>
+  <String Id="FilesInUseOkButton">&amp;OK</String>
+  <String Id="FilesInUseCancelButton">&amp;Cancel</String>
+  <String Id="ChangeLogLink">&lt;a href="https://www.id.ee/en/article/id-software-versions-info-release-notes/"&gt;Changelog&lt;/a&gt;</String>
```

Comment:
This needs to be discussed with Ian. But I will accept the merge request as-is for now.